src/umpire/util/Macros.hpp

```diff
@@ -126,4 +126,10 @@
 #define UMPIRE_UNLOCK \
   m_mutex->unlock();
 
+#define UMPIRE_CHECK_ALLOCATOR(record, name) \
+  if (record->m_strategy != this) { \
+    UMPIRE_ERROR(ptr << " was not allocated by " << m_allocator->getName()); \
```

Comment:
I think you want this to be:
```suggestion
    UMPIRE_ERROR(ptr << " was not allocated by " << name); \
```

Reply:
I was rushing, thanks for the correction.